features/fixtures/delayed_job/Dockerfile

```diff
@@ -1,12 +1,18 @@
 ARG ruby_version
 FROM ruby:$ruby_version
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+WORKDIR /bugsnag
+COPY *.gem ./
+RUN gem unpack bugsnag*.gem /bugsnag
```

Comment:
Doing this means that you always have to rebuild the docker container I think. Perhaps unpacking outside the container and pulling that into the directory works better?